packages/shared/src/utils/route/route.ts

```diff
@@ -1,5 +1,5 @@
 // Checks if pathname matches route. (Works even with query string /?)
-
+import React from 'react';
```

Comment:
we are not exporting any components here as far as I see
why do we need react?

Reply:
We are using the `React.ElementType`, because of that we imported that